### PR TITLE
Add wait for alt address account delation

### PR DESCRIPTION
**Pull request by lyubenkov**

### CODE REVIEW

File: integration/tests/economy/test_economics.py [hunk 1]
```diff
@@ -765,9 +765,19 @@ def test_deploy_big_contract_with_structures(self, client_and_price, account_wit
         )
         get_gas_used_percent(w3_client, receipt)
 
-    @pytest.mark.parametrize("value", [20, 25, 55])
+    @pytest.mark.slow
+    @pytest.mark.timeout(30 * Time.MINUTE)
+    @pytest.mark.parametrize("value", [25, 55])
```
Comment (kristinaNikolaevaa) [inline integration/tests/economy/test_economics.py]: Did you remove test with value=20?

File: integration/tests/economy/test_economics.py [hunk 2]
```diff
@@ -785,6 +795,26 @@ def test_call_contract_with_mapping_updating(
         assert receipt["status"] == 1
         wait_condition(lambda: sol_balance_before != operator.get_solana_balance())
 
+        solana_trx = web3_client.get_solana_trx_by_neon(receipt["transactionHash"].hex())
+        sol_trx_with_alt = None
+
+        for trx in solana_trx["result"]:
```
Comment (kristinaNikolaevaa) [inline integration/tests/economy/test_economics.py]: Looks like this part is copied from another test. Maybe it's better to move common part to some function?

File: integration/tests/economy/test_economics.py (hunk 1 above)
Comment (lyubenkov) [reply integration/tests/economy/test_economics.py]: Restored

File: integration/tests/economy/test_economics.py (hunk 2 above)
Comment (lyubenkov) [reply integration/tests/economy/test_economics.py]: Moved to separate function

File: utils/solana_client.py [hunk 3]
```diff
@@ -195,3 +195,13 @@ def wait_transaction(self, tx):
         except TimeoutError:
             return None
         return self.get_transaction(Signature.from_string(tx), max_supported_transaction_version=0)
+
+    def account_deleted(self, account_address) -> bool:
```
Comment (gigimon) [inline utils/solana_client.py]: rename pls to account_is_exist, it's more relevant name

**Review by gigimon — APPROVED** [review]

**Comment by kristinaNikolaevaa** [comment]
https://github.com/neonlabsorg/neon-tests/actions/runs/8001676220/job/21853475830#step:5:161 @lyubenkov all 3 tests are fail  on this branch

File: utils/solana_client.py (hunk 3 above)
Comment (lyubenkov) [reply utils/solana_client.py]: Renamed to account_exists, as `is exist` is  grammatical error concerning verb usage and conjugation.

**Comment by gigimon** [comment]
@lyubenkov https://github.com/neonlabsorg/neon-tests/actions/runs/8046698951/job/21974621302

**Comment by lyubenkov** [comment]
@gigimon after additional changes it's passing https://github.com/neonlabsorg/neon-tests/actions/runs/8052428917/job/21992720281

**Pull request merged**